packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -23,58 +23,25 @@ export class FormSchemaService {
     private translate: TranslateService,
   ) {}
 
-  public getFormTranslationsByUuid(formUuid: string, cached: boolean = true) {
-    const cachedCompiledTranslations = this.getCachedTranslationsByUuid(formUuid);
+  public getFormTranslationsByUuid(formUuid: string) {
     const currentLang = (window as Window).i18next?.language.substring(0, 2).toLowerCase() || 'en';
     this.translate.setDefaultLang(currentLang);
 
-    console.log(cachedCompiledTranslations, 'cached');
-    if (cachedCompiledTranslations && cached) {
-      cachedCompiledTranslations.translations.forEach((translationResourceUuid: string) => {
-        const translationResource = this.getCachedTranslationsByUuid(translationResourceUuid);
-        if (translationResource) {
-          this.translate.setTranslation(translationResource.language, translationResource.translations, true);
-        }
-      });
-
-      return of(cachedCompiledTranslations);
-    }
-
-    return forkJoin({
-      unCompiledTranslations: this.getFormTranslationsByUuidFromServer(formUuid).pipe(take(1)),
-      formMetadataObject: this.formsResourceService.getFormMetaDataByUuid(formUuid).pipe(take(1)),
-    }).pipe(
-      map(({ unCompiledTranslations, formMetadataObject }) => {
-        const formTranslations: any = unCompiledTranslations.form;
-
-        if (formTranslations?.translations) {
-          formTranslations.translations.forEach((translationUuid: string) => {
-            this.getFormTranslationResourceByUuidFromServer(translationUuid)
-              .then((result) => {
-                this.cacheTranslationsByUuid(translationUuid, result);
-                this.translate.setTranslation(result.language, result.translations, true);
-              })
-              .catch((error) => {
-                console.error(`Error getting AMPATH translation resource: ${error.display}.`);
-              });
-          });
-        }
+    const formMetadataObject: any = this.formsResourceService.getFormMetaDataByUuid(formUuid).pipe(take(1));
```

Comment:
Also, it would be much better if we could just use the form metadata we're already loading [over here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/7bc9d5f6f86b62822f5ba4f4bbd41c635719ed85/packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts#L24). There's no constraint that says we need to use this function with this signature and we should only be loading the form object once from the server if at all possible.